test/static_fixtures/facts/centos8-legacy-facter-2.5.json

```diff
@@ -0,0 +1,118 @@
+{
```

Comment:
Please rename this. The CentOS project prefers CentOS Linux vs CentOS Stream to differentiate the two.